src/ucp/wireup/wireup_cm.c

```diff
@@ -214,8 +214,7 @@ static void uct_wireup_cm_tmp_ep_cleanup(ucp_ep_h tmp_ep, ucs_queue_head_t *queu
     }
 
     ucs_trace("deleting tmp_ep %p", tmp_ep);
-    ucp_ep_delete(tmp_ep);
-    tmp_ep = NULL;
+    ucp_ep_destroy_base(tmp_ep);
```

Comment:
@evgeny-leksikov @brminich why in some other case you said we could not use ucp_ep_destroy_base() for tmp_ep since we can have lanes on it?

Reply:
in case of error in the middle of wireup, tmp_ep has wireup lanes which should be destroyed